### PR TITLE
Location streaming

### PR DESCRIPTION
**Pull request by nicodh**

### CODE REVIEW

File: binding.gyp [hunk 1]
```diff
@@ -22,7 +22,7 @@
           "libraries": [
             "../deltachat-core/builddir/src/libdeltachat.a",
             "/usr/local/Cellar/libetpan/1.9.2_1/lib/libetpan.a",
-            "../deltachat-core/builddir/libs/netpgp/libnetpgp.a",
+            "/usr/local/lib/rpgp/libpgp_ffi.a",
```
Comment (Jikstra) [inline binding.gyp]: We need to change this for packaging but should be fine for the moment.

File: scripts/rebuild-core.js [hunk 2]
```diff
@@ -33,6 +34,11 @@ spawn('ninja', verbose ? [ '-v' ] : [], {
   stdio: 'inherit'
 })
 
+spawn('npx', [ 'node-gyp', 'rebuild' ], {
+  cwd: path.resolve(__dirname, '../'),
+  stdio: 'inherit'
+})
+
```
Comment (Jikstra) [inline scripts/rebuild-core.js]: rebuilding core and rebuilding bindings should be done in package.json

File: test.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+  const a = binding.dcn_get_locations(dc.dcn_context, 255, 10)
+  const b = binding.dcn_array_get_cnt(a)
+  const c = binding.dcn_array_get_id(a, 0)
+  const d = binding.dcn_array_get_accuracy(a, 2)
+  console.log(a, b, c, d)
+}
+
+setup_dc(main)
+
```
Comment (Jikstra) [inline test.js]: We can throw out that file again, maybe include something like this in the readme (at least the setup_dc/main boilerplate)

**Comment by nicodh** [comment]
This can be merged, but has a dependency on location-stream branch of deltachat-node

**Pull request merged**